### DISCUSSION TITLE
LAB 04.02 - BUILDING DATASETS.

### DISCUSSION THREAD

**Discussion by joanfco30** [rramosp/ai4eng.v1#120] [04 - GESTIÓN DE DATOS]
Buenas noches un cordial saludo.
Tengo unas inquietudes con el LAB 04.02 - BUILDING DATASETS y me gustaría tener alguna realimentación para continuar este trabajo. 
Hasta el momento he realizado los siguiente: 
Construyo el data set con la información solicitada.
![image](https://user-images.githubusercontent.com/60493083/163814057-d3f6ea7c-c167-4c7c-9fc9-1e34f32afec2.png)
Yo realizo la construcción del data set de la siguiente manera: 
![image](https://user-images.githubusercontent.com/60493083/163814176-2a8f6719-e952-4dfa-81a8-2e8a2f1fc30c.png)
 En esta etapa es donde me siento perdido y no se como avanzar. 

 Quedo atento a cualquier comentario o ayuda para continuar este laboratorio. 
  Muchas gracias y un feliz día.

**Comment by Felipe-RA** [comment]
Hola joanfco30

Disculpa la tardanza en responderte.

Sería de mucha ayuda que profundizaras un poco más en qué te sientes perdido exactamente, intenta ser un poco más específico. ¿Qué es lo que consideras que no te permite avanzar?

**Comment by joanfco30** [comment]
Buenas tardes Andres, muchas gracias ya logre solucionar. Tuve vario inconvenientes, especialmente un error de compresión lectora ya que estaba evaluando sin decimales. 
Muchas gracias
